k8s_handle/templating.py

```diff
@@ -113,6 +116,40 @@ def generate_by_context(self, context):
                 raise TemplateRenderingError('Unable to render {}, due to: {}'.format(template, e))
         return output
 
+    def _filter_tagged_templates(self, templates, only_tags, skip_tags):
+        return [i for i in templates if self._evaluate_tags(i, only_tags, skip_tags)]
+
+    def _evaluate_tags(self, template, only_tags, skip_tags):
```

Comment:
The method return `true` or `false`, maybe `_is_template_tagged(...)` more accurate name?

Reply:
I don't like both names actually (especially after moving "template" part into separate function). :slightly_frowning_face:

Please let me know if `_is_template_tagged(self, tags, only_tags, skip_tags)` still looks better for you after last changes.